### PR TITLE
Add uk.co.wagemaker.SmallTextPad

### PR DESCRIPTION
**Pull request by gcclinux**
Please help me convert this to a flatpak and upload it into the store :-)

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build uk.co.wagemaker.SmallTextPad

File: uk.co.wagemaker.SmallTextPad.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+					"type": "file",
+					"path": "uk.co.wagemaker.SmallTextPad.appdata.xml"
+				},
+				{
+					"type": "file",
+					"path": "run_smalltextpad.sh"
+				},
+				{
+					"type": "file",
+					"path": "SmallTextPad.jar"
```
Comment (barthalion) [inline uk.co.wagemaker.SmallTextPad.json]: You shouldn't store the jar file here.

File: run_smalltextpad.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (barthalion) [inline run_smalltextpad.sh]: Do you actually need it? You control the environment so it makes no sense to run these checks (and export DISPLAY),

File: README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# flathub
+Pull requests for new applications to be added
+# Simple to build <br>
+$ flatpak-builder --repo=repo --force-clean build-dir uk.co.wagemaker.SmallTextPad.json <br>
```
Comment (barthalion) [inline README.md]: Please don't duplicate documentation.

**Comment by gcclinux** [comment]
Hello,

Most likely not but it's the script is use on other packages this will be my first flatpak working properly if I get that far 👍 

Thank for your help

⁣Sent from BlueMail ​

**Comment by barthalion** [comment]
I'm closing it due to inactivity – but please re-open it or resubmit as new pull request if you would like to see it merged in the future.

**Pull request closed without merging**